tests/installation/confirm_installation_maintenance.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: This test module confirms the popup for installation and additionally the license popup in case it appears.
+
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base 'y2_installbase';
+
+sub run {
+    my $install_popup = $testapi::distri->get_ok_popup_controller();
+    my $license_popup = $testapi::distri->get_license_popup();
+    # First check is for the confirm installation popup in order to save time as
```

Comment:
we could include the example of the Java popup, if you have saved some screenshot, to be more specific, although the solution is generic, which I don't like much, because I would like that we have control of each license, but ok, for the sake of not fixing the test and not need to fix the automation, let's assume that any non-free license at the end introduced as addon should be accepted. (actually you could also say that, that this happens only when when we introduce a maintenance update that has non-free licence as an additional addon/product during the installation).

Reply:
I can change the comments to make it clear that any license popup will be accepted here.